Gemfile

```diff
@@ -2,7 +2,9 @@ source 'https://rubygems.org'
 
 gemspec
 
+# gem 'byebug', '<= 9.0.6'
```

Comment:
Commented code always looks ugly, for convenience, please consider introducing `Gemfile.local.rb` instead